config/locales/en.yml

```diff
@@ -327,3 +327,9 @@ en:
           failed: |-
             You no not have enough vCPU quota to launch Rancher's cluster in the *%{region}* region. Please request a quota increase of %{vcpu_count} vCPUs for *%{instance_family}*.
             [More Info](https://learn.microsoft.com/en-us/azure/quotas/per-vm-quota-requests#request-an-increase-for-adjustable-quotas)
+        azure_public_ip_address_quota:
+          title: "Azure Public IP Address Quota"
+          passed: "You have room in your quota for Rancher's public IP address."
+          failed: |-
+            You no not have a public ip address available in the *%{region}* region. Please release a public IP address, switch to a different region, or request a quota increase.
```

Comment:
s/You no not/You do not
(probably copypasta from previous text)

Reply:
Fixed in 78c019c